test/features/pagecontent/revisions.js

```diff
@@ -167,6 +167,19 @@ describe('revision requests with en.wikipedia.org', function() {
             assert.deepEqual(res.status, 403);
         });
     });
+
+    it('should restrict user and comment', function() {
+        return preq.get({
+            uri: server.config.bucketURL + '/title/User:Pchelolo%2fRestricted_Rev'
+        })
+        .then(function(res) {
+            assert.deepEqual(res.status, 200);
+            var item = res.body.items[0];
+            assert.deepEqual(item.user_id, null);
+            assert.deepEqual(item.user_text, null);
+            assert.deepEqual(item.comment, null);
```

Comment:
For all of the three checks above: better to have:

```js
assert.deepEqual(!!item.comment, false);
```

so that the check passes regardless of the false-y value the field is set to (`null`, `undefined`, `''`, etc)